portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFolderLocalServiceImpl.java

```diff
@@ -1463,6 +1422,81 @@ protected void validateFolderName(String folderName)
 		}
 	}
 
+	private void _updateWorkflowDefinitionLinks(
+			long folderId, List<Long> fileEntryTypeIds, int restrictionType,
+			ServiceContext serviceContext, Set<Long> originalFileEntryTypeIds)
+		throws PortalException {
+
+		if (!GetterUtil.getBoolean(
```

Comment:
I thing that we are breaking api for the customer that are using local api. Since, the default value is false and for that reason they can't update workflow definitions

Reply:
same @Mikellorza, default is true, not false